network/rfc004/bbolt_publisher.go

```diff
@@ -0,0 +1,40 @@
+package rfc004
+
+import (
+	"go.etcd.io/bbolt"
+)
+
+type bboltPublisher struct {
```

Comment:
the name is a bit weird since bbolt is storage not a publisher?

Reply:
It uses a bbolt bucket internal to the BBolt DAG implementation. The whole thing is a bit ugh and "just make it work", so it needs to be revamped anyways. But I'd like to separate that work to avoid large scope creep (https://github.com/nuts-foundation/nuts-node/issues/8)